### PR TITLE
Remove statik package installation procedure in contributing page, re… (#2998)

### LINKED ISSUE #2998: Documentation - Fix statik package installation procedure in contributing page
The procedure to install statik (go get) was deprecated since go 1.17:

> % go get github.com/rakyll/statik                                                                                                                                                                   
go: downloading github.com/rakyll/statik v0.1.7
go get: installing executables with 'go get' in module mode is deprecated.
	Use 'go install pkg@version' instead.
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.

For go 1.17 and above use:

> go install github.com/rakyll/statik@&lt;version&gt;

### PR DESCRIPTION
**Pull request by N-o-Z**
In go version 1.17 command 'go get' is deprecated.
Edit Makefile and modify all go get commands to use go install instead.
Remove statik package installation procedure in contributing doc as it is installed as part of the make build.
fixes #2998

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -74,12 +74,12 @@ clean:
 check-licenses: check-licenses-go-mod check-licenses-npm
 
 check-licenses-go-mod:
-	go get github.com/google/go-licenses
+	go install github.com/google/go-licenses@latest
```
Comment (nopcoder) [inline Makefile:77]: can you use $(GOBIN) instead of calling go directly

**Review by nopcoder — APPROVED** [review]

**Review by nopcoder — CHANGES_REQUESTED** [review]
Just update the `go` command to use the variable instead.

File: Makefile (hunk 1 above)
Comment (N-o-Z) [reply Makefile]: Assume you meant $(GOCMD)

**Review by nopcoder — APPROVED** [review]
LGTM

**Pull request merged**